lib/Server.js

```diff
@@ -3254,9 +3254,36 @@ class Server {
    * @param {string | string[]} watchPath
    * @param {WatchOptions} [watchOptions]
    */
-  watchFiles(watchPath, watchOptions) {
+  watchFiles(watchPath, watchOptions = {}) {
     const chokidar = require("chokidar");
-    const watcher = chokidar.watch(watchPath, watchOptions);
+    const isGlob = require("is-glob");
```

Comment:
What do you think about passing parameters here? Before we have them in `watchOptions` for globs